BraintreeCore/src/main/java/com/braintreepayments/api/PaymentMethodClient.java

```diff
@@ -26,18 +31,46 @@ public PaymentMethodClient(BraintreeClient braintreeClient) {
         this.braintreeClient = braintreeClient;
     }
 
+    /**
+     * Parses a response from the Braintree gateway for a list of payment method nonces.
+     *
+     * @param jsonBody Json-formatted String containing a list of {@link PaymentMethodNonce}s
+     * @return List of {@link PaymentMethodNonce}s contained in jsonBody
+     * @throws JSONException if parsing fails
+     */
+    private static List<PaymentMethodNonce> parsePaymentMethodNonces(String jsonBody) throws JSONException {
+        JSONArray paymentMethods =
+            new JSONObject(jsonBody).getJSONArray(PAYMENT_METHOD_NONCE_COLLECTION_KEY);
+
+        if (paymentMethods == null) {
+            return Collections.emptyList();
+        }
+
+        List<PaymentMethodNonce> paymentMethodNonces = new ArrayList<>();
+        JSONObject json;
+        PaymentMethodNonce paymentMethodNonce;
+        for(int i = 0; i < paymentMethods.length(); i++) {
+            json = paymentMethods.getJSONObject(i);
+            // TODO: leverage compileOnly gradle dependencies to return strongly-typed nonces (e.g. CardNonce, PayPalAccountNonce etc.)
+            paymentMethodNonce = PaymentMethodNonce.fromJSON(json);
+            if (paymentMethodNonce.getType() != PaymentMethodType.GOOGLE_PAY) {
```

Comment:
Why are we filtering out Google Pay nonces?

Reply:
🤷‍♂️  we kept the logic the same. Here's the [OG test](https://github.com/braintree/braintree_android/blob/23f48a492fbb7f1b900ccc0b2a44c9d7512f5add/BraintreeCore/src/test/java/com/braintreepayments/api/PaymentMethodUnitTest.java#L119).

Reply:
OK, sounds good!